### PR TITLE
Add batch page

### PR DESCRIPTION
**Pull request by Sreelatha-Vallabhaneni**
I created an endpoint to get id of the user by UID, which is needed to create the batch.

### CODE REVIEW

**Comment by gizemcandemir** [comment]
Hi @zkwsk, it worked with me :thinking_face: I mean I can see the page `http://localhost:3000/addbatch` when I'm logged in with ningry0@csmonitor.com.

File: src/client/containers/add-batch-page/add-batch-page.component.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-import AddBatchImage from '../../components/add-batch-image/add-batch-image.component';
 import Footer from '../../components/footer/footer.component';
+import './add-batch-page.style.css';
 
 const AddBatchPage = () => {
   return (
     <>
-      <div>
+      <div className="batch-page-nav">
         <SidebarMenu isActive={true} isUser={true} />
```
Comment (gizemcandemir) [inline src/client/containers/add-batch-page/add-batch-page.component.js:11]: `isActive` props should be `false` for Add Batch page (it's only true for Chart Detail Page).

File: src/client/App.js [hunk 2]
```diff
@@ -39,6 +40,7 @@ function App() {
             component={ForgotPassword}
           />
           <PrivateRoute exact path="/dashboard" component={Dashboard} />
+          <PrivateRoute exact path="/addbatch" component={AddBatch} />
```
Comment (gizemcandemir) [inline src/client/App.js:48]: I would write the path as `"/add-batch"` with the dash between the words.

Comment (dpfernandes) [reply src/client/App.js:48]: Yes please use add-batch.

It's already working like that on the side bar menu

image

**Comment by Sreelatha-Vallabhaneni** [comment]
Hi @zkwsk  I fixed that 401 issue and other issues which were reviewed by @gizemcandemir  (i fixed all the issues) but i have a git problem it won't allow me to push it. i tried in many ways and @cecastosic also tried to help ....it didn't work out.......plz help me to fix it

**Comment by Sreelatha-Vallabhaneni** [comment]
I am closing this PR #198 because I have created a new PR #207 for ADD BATCH

**Pull request closed without merging**